admin-dev/themes/new-theme/.webpack/dev.js

```diff
@@ -11,10 +15,62 @@ function devConfig() {
     {
       devtool: 'inline-source-map',
       devServer: {
+        client: {
+          logging: 'error',
+          progress: false,
+          overlay: {
+            errors: true,
+            warnings: false,
+          },
+        },
         hot: true,
-        contentBase: path.resolve(__dirname, '/../public'),
-        publicPath: '/',
+        static: {
+          directory: path.join(__dirname, '/../public'),
+          watch: false,
+        },
+        port: 3000,
+        watchFiles: [
+          path.join(__dirname, '/../**/*.tpl'),
```

Comment:
What about vue or ts files?

Reply:
watchFiles it not needed tbh, I removed it